util/declarations/index.mjs

```diff
@@ -44,7 +44,7 @@ const run = ({
   // Important file/directory locations
   const declarationsFile = path.join(outputFile)
   const schemasFolder = path.join(srcFolderArg, 'schemas')
-  const sharedSchemasFolder = path.join(__dirname, '..', '..', 'src', 'schemas')
+  const sharedSchemasFolder = path.join(__dirname, '..', '..', 'node_modules', '@firebolt-js', 'schemas', 'src', 'schemas')
```

Comment:
We should make a todo to parameterize this, so the sharedSchemasFolder is passed in. We shouldn't assume everyone using the tool has this dependency.